include/picongpu/plugins/adios/WriteMeta.hpp

```diff
@@ -189,7 +189,7 @@ namespace writeMeta
              *    3D: z-lower, z-upper, y-lower, y-upper, x-lower, x-upper
              *    2D: y-lower, y-upper, x-lower, x-upper
              */
-            GetStringProperties<FieldManipulator> fieldBoundaryProp;
+            GetStringProperties<absorber::ExponentialDamping> fieldBoundaryProp;
```

Comment:
TODO for later: here we always write data of the exponential damping absorber, even if PML is used.